articles/quantum-InstallConfig.md

```diff
@@ -126,3 +132,15 @@ In this section you will clone the quantum samples & libraries repository, and r
           Teleportation successful!!
   ```
 
+
+### Optional Dependencies ###
+
+Some of the samples demonstrate using the Quantum Development Kit with other programming environments, and have additional installation requirement:
+
+- Some samples use the Node.js Package Manager (NPM) to handle user interface dependencies.
+  NPM can be installed [manually](https://nodejs.org/en/download/current/) or through a [package manager](https://nodejs.org/en/download/package-manager/).
+- The Python interoperability feature has been developed for the [Anaconda distribution](https://www.anaconda.com/download/) of Python 3.6.
+  Please see the [README](https://github.com/Microsoft/Quantum/tree/master/Samples/PythonInterop/README.md) file provided with the Python sample for more details.
+
+Additionally, the Quantum Development Kit is provided along with libraries for non-commercial use.
```

Comment:
Is QDK actually provided with these libraries, or are these libraries provided separately? I suspect there is a line to be drawn here. I'd say something like "You can find more Q# libraries for non-commercial use at ..."

Reply:
Since the non-commercial phase estimation operations are documented with the rest of the QDK APIs, I think it makes sense to keep the phrasing as is for now, with an eye to revisiting as you suggest.